core/osm_connector.py

```diff
@@ -314,7 +338,7 @@ def _build_stop(self, stop, stop_type):
 
         # Make sure name is not empty
         if 'name' not in stop.tags:
-            stop.tags['name'] = Stop.NO_NAME
+            stop.tags['name'] = "[" + self.stop_no_name + "]"
```

Comment:
Yes, it is! Because I don't want to hit the nearest search if it was unsuccessful once. This is the same behaviour as before.

When creating the stops from the first query the ones without name get something like `[No name]`, then it tries to guess their name, in case it was not possible to find one the name will be `No name` (without brackets) and next time it would not search (and query overpass api) again.